app/views/jobs/show.html.erb

```diff
@@ -10,4 +10,10 @@ Level of Interest: <%= @job.level_of_interest %>
 City: <%= @job.city %>
 <br>
 
+Tags:
+<p><% @job.tags.uniq.each do |tag| %>
+  <%= tag.name %></br>
+<% end %></p>
+
```

Comment:
Using `.uniq` here is a bit of a code smell. Makes me think you'd allow duplicate tags per job, which could then alter results of how many jobs each tag has and the average salary per tag...